src/modules/wallets/ExplorerWallet.js

```diff
@@ -529,18 +524,18 @@ class ExplorerWallet {
 
   /**
    * Create and send a FLO tx with a custom output
-   * @param {object|Array.<object>} outputs
```

Comment:
If I remember correctly an output object was
```
{ address: xxxx. value: xxxx } 
```
With these changes are you still allowed to specify multiple output addresses and amounts?

Reply:
Multiple output addresses should still be allowed, it was likely just a documentation merge issue :) 

Fixed in latest commit!